src/backend.github.js

```diff
@@ -191,20 +188,21 @@ var _ = Mavo.Backend.register($.Class({
 
 		if (this.notice) {
 			lastNoticeName = this.notice.o.name;
-			this.notice.nonAnimatedClose();
+			this.notice.o.closeTransition = false;
```

Comment:
I’m a little confused, I see you added a `style` parameter, why do you also need `closeTransition`?